### PR TITLE
5 个简单步骤为您的网站创建 Sitemap

### LINKED ISSUE #6590: 5 个简单步骤为您的网站创建 Sitemap
* 原文链接：[5 Easy Steps to Creating a Sitemap For a Website](https://www.quicksprout.com/creating-website-sitemap/)
* Markdown文件：[文件地址](https://github.com/xitu/gold-miner/blob/master/TODO1/creating-website-sitemap.md)
* PR 地址：
* 文章分类：**前端**
* **注意：文件位置在 `TODO1` 文件夹中，不是之前的 `TODO` 了。**
----
* 翻译时间： 天
* 校对时间： 天
* 翻译积分： 分
* 校对积分： 分
* [积分有什么用](https://github.com/xitu/gold-miner/wiki)

### PR DESCRIPTION
**Pull request by cyz980908**
5 个简单步骤为您的网站创建 Sitemap

译文翻译完成，resolve #6590

### CODE REVIEW

**Comment by Chorer** [comment]
校对认领

**Comment by Chorer** [comment]
校对认领 @Glowin

**Comment by fanyijihua** [comment]
@Chorer 好的呢 :beer:

File: TODO1/creating-website-sitemap.md [hunk 1]
```diff
@@ -2,202 +2,202 @@
...
+话是这样说，但是我是一个遵循成功网站和企业最佳实践的坚定信仰者。所以，如果像 Apple 这样的大公司将 sitemap 写到 robots.txt 页面，这或许对我们来说是一个不错的主意。
 
-### Step 5: Submit your sitemap
+### 第 5 步：提交 sitemap 
 
-Now that your sitemap has been created and added to your site files, it’s time to submit them to search engines.
+现在您的 sitemap 已经创建好并添加到您的站点文件中了，是时候将它们提交给搜索引擎了。
 
-In order to do this, you need to go through [Google Search Console](https://search.google.com/search-console/about). Some of you may already have this set up. If not, you can get started very easily.
+您需要通过 [Google Search Console](https://search.google.com/search-console/about)来提交。有些人可能已经使用过它了。就算没有，您也可以很容易地就上手它。
```
[the remark this answers is not in the document]
Comment (Chorer) [reply TODO1/creating-website-sitemap.md]: 很容易地就上手它。=> 很容易地就上手。

一般说“快速上手”，后面应该不能接宾语构成动宾结构

File: TODO1/creating-website-sitemap.md [hunk 2]
```diff
@@ -2,202 +2,202 @@
...
-Once you’re on the search console dashboard, navigate to Crawl > Sitemaps.
+进入 Search Console 控制面板后，导航至 Crawl > Sitemaps。
 
 ![Google search console](https://quicksprout-wpengine.netdna-ssl.com/wp-content/uploads/2019/01/google-search-console.png)
 
-Next, click on Add/Test Sitemap on the top right corner of the screen.
+接着，单击屏幕右上角的 Add/Test Sitemap。
 
-This is a chance for you to test your sitemap again for any errors before you continue. Obviously, you’ll want to fix any mistakes found. Once your sitemap is free of errors, click submit and that’s it. Google will handle everything else from here. Now crawlers will index your site with ease, which will boost your SEO ranking.
+这是您在之前测试 sitemap 之后又一次检验是否有 bug 的机会。显然，您需要修复所发现的任何错误。一旦您的 sitemap 没有错误，点击提交就可以了。Google 将处理这一切。现在爬虫将很容易地索引您的网站，这将提高您的 SEO 排名。
```
Comment (Chorer) [inline TODO1/creating-website-sitemap.md]: 这是您在之前测试 sitemap 之后又一次检验是否有 bug 的机会。
=>
这里可以让您在继续下一步之前再次检验 sitemap 是否有错误。

File: TODO1/creating-website-sitemap.md [hunk 3]
```diff
@@ -2,202 +2,202 @@
...
 
-## Alternative options
+## 代替方案
 
-While these five steps are pretty simple and straightforward, some of you might be a little uncomfortable manually changing the code on your website. That’s perfectly understandable. Fortunately for you, there are plenty of other solutions that can create a sitemap for you, without having to edit the code yourself.
+虽然这五个步骤非常简单明了，但是可能还是会有些人对手动修改网站上的代码感到不舒服。这完全可以理解。幸运的是，还有许多其他的解决方案可以为您创建 sitemap，并且您不用自己编辑代码。
```
Comment (Chorer) [inline TODO1/creating-website-sitemap.md]: 其他的 => 其它的

Comment (Chorer) [reply TODO1/creating-website-sitemap.md]: 还有许多其他的解决方案可以为您创建 sitemap，并且您不用自己编辑代码。
=>
还有许多其它的解决方案可以让您不用自己编辑代码也能创建 sitemap。

File: TODO1/creating-website-sitemap.md [hunk 4]
```diff
@@ -2,202 +2,202 @@
...
 
-I’ll go through some of the top options for you to consider.
+我将介绍一些最常用的代替方案供您考虑。
 
-### Yoast plugin
+### 插件 Yoast
```
Comment (Chorer) [inline TODO1/creating-website-sitemap.md]: 插件 Yoast => Yoast 插件

File: TODO1/creating-website-sitemap.md [hunk 5]
```diff
@@ -2,202 +2,202 @@
...
 
-If you have a WordPress website, you can install the [Yoast plugin](https://kb.yoast.com/kb/enable-xml-sitemaps-in-the-wordpress-seo-plugin/) to create a sitemap for your website.
+如果你有一个 WordPress 网站，你可以安装 [Yoast 插件](https://kb.yoast.com/kb/enable-xml-sitemaps-in-the-wordpress-seo-plugin/)来为你的网站创建 sitemap。
```
Comment (Chorer) [inline TODO1/creating-website-sitemap.md]: 这一段的“你”全部改为“您”

File: TODO1/creating-website-sitemap.md [hunk 6]
```diff
@@ -2,202 +2,202 @@
...
 
-Yoast gives you the option to turn your sitemap on and off with a simple toggle switch. You can find all of your XML sitemap options from the SEO tab via WordPress once the plugin has been installed.
+Yoast使您可以选择通过简单的拨动开关来打开和关闭 sitemap。插件安装好后，您可以从 WordPress 的 SEO 标签页中找到所有 XML sitemap 选项。
```
Comment (Chorer) [inline TODO1/creating-website-sitemap.md]: Yoast使您可以选择通过简单的拨动开关来打开和关闭 sitemap
=>
Yoast 可以让您通过简单的拨动开关来打开和关闭 sitemap

File: TODO1/creating-website-sitemap.md [hunk 7]
```diff
@@ -2,202 +2,202 @@
...
 
 ### Screaming Frog
 
-[Screaming Frog](https://www.screamingfrog.co.uk/xml-sitemap-generator/) is desktop software that offers a wide range of SEO tools. It’s free to use and generate a sitemap as long as the website has fewer than 500 pages. For those of you with larger websites, you’ll need to upgrade the paid version.
+[Screaming Frog](https://www.screamingfrog.co.uk/xml-sitemap-generator/) 是一款桌面软件，它提供了广泛的搜索引擎优化工具。只要网站的页面不超过 500 页，您就可以免费使用和生成 sitemap。对于那些拥有大型网站的用户，您就需要升级付费版本了。
```
Comment (Chorer) [inline TODO1/creating-website-sitemap.md]: 它提供了广泛的搜索引擎优化工具 
=>
它提供了大量的 SEO 工具

前面都是没有将 SEO 翻译出来的，建议这里不译，保持翻译风格一致

File: TODO1/creating-website-sitemap.md [hunk 8]
```diff
@@ -2,202 +2,202 @@
...
 
-Screaming Frog allows you to make all of the coding changes that we talked about earlier, but without actually changing the code yourself. Instead, you follow a prompt that’s much more user-friendly, and written in plain English. Then the code for the sitemap file will be changed automatically. Here’s a screenshot to show you what I mean.
+Screaming Frog 支持我们前面讨论过的所有代码的更改，而且不需要您自己实际编写代码。相反，您会看到一个更友好的提示，而且是用简单的英语写的。然后 sitemap 文件的代码将自动更改。这里有个截屏会告诉你我在说什么。
```
Comment (Chorer) [inline TODO1/creating-website-sitemap.md]: 相反，您会看到一个更友好的提示，而且是用简单的英语写的。
=>
相反，您根据提示操作就行了，这个提示更友好，而且是用通俗易懂的英语写的。

Comment (Chorer) [reply TODO1/creating-website-sitemap.md]: 这里有个截屏会告诉你我在说什么。
=>
下面的截图就是我要表达的意思。

File: TODO1/creating-website-sitemap.md [hunk 9]
```diff
@@ -2,202 +2,202 @@
...
-Just navigate through the tabs, change your settings, and the sitemap file will be adjusted accordingly.
+只需选择标签页，更改设置，sitemap 文件就会相应地进行调整。
 
 ### Slickplan
 
-I really like Slickplan because of the visual sitemap builder feature. You’ll have the opportunity to use a sitemap template, similar to the ones we looked at earlier.
+我非常喜欢 Slickplan，因为它有可视化的 sitemap 构建功能。您将有机会使用 sitemap 模板，类似于我们前面看到的网站结构模板。
 
-From here, you can drag and drop different pages into the template to organize the structure of your website. Once you’re done, and you’re happy with the way your visual sitemap looks, you can export it as an XML file.
+从这里，您可以拖放不同的页面到模板来组织您的网站结构。完成后，如果您对可视化的 sitemap 感到满意，就可以将其导出为 XML 文件。
```
Comment (Chorer) [inline TODO1/creating-website-sitemap.md]: 从这里 => 在这里

File: TODO1/creating-website-sitemap.md [hunk 10]
```diff
@@ -2,202 +2,202 @@
...
 
-Slickplan is paid software, but they offer a free trial. It’s at least worth trying if you’re on the fence about purchasing a plan.
+Slickplan 是付费软件，但它们提供免费试用。如果您在犹豫是否购买它，可以去试试。
 
-## Conclusion
+## 总结
 
-If you’re ready to take your SEO strategy to the next level, you need to create a sitemap for your website.
+如果您打算提升一下您的 SEO 策略，您需要做的就是为您的网点创建一个 sitemap。
```
Comment (Chorer) [inline TODO1/creating-website-sitemap.md]: 网点 => 站点

File: TODO1/creating-website-sitemap.md [hunk 11]
```diff
@@ -2,202 +2,202 @@
...
 
-There is no reason to be intimidated by this anymore. As you can see from this guide, it’s easy to create a sitemap in just five steps.
+有了这篇指南，对您而言 sitemap 就再也不是难题了，只需五个步骤即可轻松创建 sitemap。
```
Comment (Chorer) [inline TODO1/creating-website-sitemap.md]: 有了这篇指南，对您而言 sitemap 就再也不是难题了，只需五个步骤即可轻松创建 sitemap。
=>
对您而言 sitemap 再也不是难题了。因为正如这篇指南所介绍的，只需五个步骤即可轻松创建 sitemap。

File: TODO1/creating-website-sitemap.md [hunk 12]
```diff
@@ -2,202 +2,202 @@
...
 
-1. Review your pages
-2. Code the URLs
-3. Validate your code
-4. Add the sitemap to the root and robots.txt
-5. Submit the sitemap
+1. 检查页面的结构
+2. 编辑 URL
+3. 验证代码的正确性
+4. 将 sitemap 添加到站点根目录 和 robots.txt
```
Comment (Chorer) [inline TODO1/creating-website-sitemap.md]: 站点根目录 和  => 站点根目录和

File: TODO1/creating-website-sitemap.md [hunk 13]
```diff
@@ -2,202 +2,202 @@
...
+5. 提交 sitemap
 
-That’s it!
+这就完事啦！
 
-For those of you who are still on the fence about manually changing code on your website, there are other options for you to consider. The Internet is full of sitemap resources, but the Yoast plugin, Screaming Frog, and Slickplan are all great choices to start.
+对于那些仍然对手动更改网站代码束手无策的那些人，您还可以选择其他代替方案。尽管互联网上有着大量的 sitemap 相关的资源，但是我推荐的 Yoast 插件，Screaming Frog 和 Slickplan 依然是很不错的选择。
```
Comment (Chorer) [inline TODO1/creating-website-sitemap.md]: 对于那些仍然对手动更改网站代码束手无策的那些人
=>
对于那些仍然对手动更改网站代码束手无策的人

Comment (Chorer) [reply TODO1/creating-website-sitemap.md]: 其他代替方案 => 其它代替方案

Comment (Chorer) [reply TODO1/creating-website-sitemap.md]: 依然是很不错的选择。=> 都是很不错的入门选择。

File: TODO1/creating-website-sitemap.md [hunk 14]
```diff
@@ -2,202 +2,202 @@
...
+参考 [Search Engine Journal 的这篇文章](https://www.searchenginejournal.com/website-structure-affects-seo/186553/)，您应该致力于创建深度较浅的 sitemap，这意味着只需单击三下即可导航到您网站上的任何页面。对于SEO而言，这会是极好的。
 
-So you need to create a hierarchy of pages based on importance and how you want them to be indexed. Prioritize your content into tiers that follow a logical hierarchy. Here’s [an example](https://blog.hubspot.com/marketing/build-sitemap-website) to show you what I’m talking about.
+因此，您需要创建一个基于页面重要级别和您具体希望它们如何被索引到的页面层次结构。这会将内容按照逻辑层次的不同划分为不同的层次。如果您不太明白，可以看看这个[例子](https://blog.hubspot.com/marketing/build-sitemap-website)。
 
-![page hierarchy](https://quicksprout-wpengine.netdna-ssl.com/wp-content/uploads/2019/01/page-hierarchy.png)
+![页面层次结构](https://quicksprout-wpengine.netdna-ssl.com/wp-content/uploads/2019/01/page-hierarchy.png)
 
-As you can see, the About page links to Our Team as well as Mission & Values. Then the Our Team page links to Management and Contact Us.
+正如您所看到的，”关于“页面（ABOUT）链接到”我们团队“页面（Our Team）以及”使命和价值观“页面（Mission & Values）。然后，”我们团队“页面（Our Team）链接到”管理部门“页面（Management）和”联系我们“页面（Contact Us）。
```
[the remark this answers is not in the document]
Comment (cyz980908) [reply TODO1/creating-website-sitemap.md]: 好的👌

File: TODO1/creating-website-sitemap.md [hunk 15]
```diff
@@ -2,202 +2,202 @@
...
+![hubspot 的 sitemap](https://quicksprout-wpengine.netdna-ssl.com/wp-content/uploads/2019/01/hubspot-sitemap.png)
 
-In addition to adding the sitemap file to your root folder, you’ll also want to add it to the robots.txt file. You’ll find this in the roots folder as well.
+除了将 sitemap 文件添加到根目录之外，您还需要将其添加到 robots.txt 文件中。您也可以在根文件夹中找到它。
 
-Basically, this to give instructions for any crawlers indexing your website.
+基本上，这可以为任何爬虫在索引您的网站时提供指导。
 
-There are a couple of different uses for the robots.txt folder. You can set this up to show search engines URLs that you don’t want them to index when they’re crawling on your site.
+robots.txt 文件有两种不同的用法。您可以对其进行设置，以显示您不希望爬虫在您的网站上搜寻时被索引到的 URL。
```
[the remark this answers is not in the document]
Comment (cyz980908) [reply TODO1/creating-website-sitemap.md]: 👍

File: TODO1/creating-website-sitemap.md [hunk 16]
```diff
@@ -2,202 +2,202 @@
...
+看看这个 [sitemap 模板](https://nationalgriefawarenessday.com/48796/website-sitemap-template)，并弄明白如何用一张表来表示您的网页。
 
-![website sitemap template](https://quicksprout-wpengine.netdna-ssl.com/wp-content/uploads/2019/01/website-sitemap-template.png)
+![网页 sitemap 模版](https://quicksprout-wpengine.netdna-ssl.com/wp-content/uploads/2019/01/website-sitemap-template.png)
 
-This is a very basic example that’s easy to follow.
+这是一个简单易懂的例子。
 
-It all starts from the homepage. Then you have to ask yourself where your homepage links to. You likely already have this figured out based on the menu options on your site.
+一切都从主页开始。然后，您访问自己主页链接到的页面。您可能已经发现，这就是您网站上的菜单选项。
```
[the remark this answers is not in the document]
Comment (cyz980908) [reply TODO1/creating-website-sitemap.md]: 👍

**Comment by cyz980908** [comment]
@Chorer 很感谢～已全部修改

**Comment by redagavin** [comment]
@Glowin 校对认领

**Comment by fanyijihua** [comment]
@redagavin 妥妥哒 :beers:

**Comment by sunui** [comment]
@redagavin 别忘了提交校对哦

**Comment by cyz980908** [comment]
@redagavin 辛苦校对，已全部修改完。

**Comment by cyz980908** [comment]
@sunui  管理员审核一下撒

**Comment by lsvih** [comment]
@cyz980908 新年快乐~

已经 merge 啦~ 快快麻溜发布到[掘金](https://juejin.im)然后给我发下链接，方便及时添加积分哟。

掘金翻译计划有自己的知乎专栏，你也可以投稿哈，推荐使用一个[好用的插件](https://chrome.google.com/webstore/detail/minimalist-markdown-edito/pghodfjepegmciihfhdipmimghiakcjf?hl=zh-CN)。
专栏地址：https://zhuanlan.zhihu.com/juejinfanyi

**Comment by cyz980908** [comment]
> @cyz980908 新年快乐~
> 
> 已经 merge 啦~ 快快麻溜发布到[掘金](https://juejin.im)然后给我发下链接，方便及时添加积分哟。
> 
> 掘金翻译计划有自己的知乎专栏，你也可以投稿哈，推荐使用一个[好用的插件](https://chrome.google.com/webstore/detail/minimalist-markdown-edito/pghodfjepegmciihfhdipmimghiakcjf?hl=zh-CN)。
> 专栏地址：https://zhuanlan.zhihu.com/juejinfanyi

新年快乐～🔗链接在这：https://juejin.im/post/5e22fc43f265da3e254c7555
